django/core/files/storage.py

```diff
@@ -328,14 +415,54 @@ def url(self, name):
         return urljoin(self.base_url, filepath_to_uri(name))
 
     def accessed_time(self, name):
+        warnings.warn(
+            'FileSystemStorage.accessed_time() is deprecated. '
+            'Use Storage.get_accessed_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         return datetime.fromtimestamp(os.path.getatime(self.path(name)))
 
     def created_time(self, name):
+        warnings.warn(
+            'FileSystemStorage.created_time() is deprecated. '
+            'Use Storage.get_created_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         return datetime.fromtimestamp(os.path.getctime(self.path(name)))
 
     def modified_time(self, name):
+        warnings.warn(
+            'FileSystemStorage.modified_time() is deprecated. '
+            'Use Storage.get_modified_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         return datetime.fromtimestamp(os.path.getmtime(self.path(name)))
 
+    def _datetime_from_timestamp(self, ts):
+        """
+        If we're running with timezone support, make an aware datetime
+        object in UTC; otherwise make a naive one in the local timezone.
+        """
+        if settings.USE_TZ:
+            return timezone.make_aware(
+                datetime.utcfromtimestamp(ts),
+                timezone.utc,
+            )
```

Comment:
Just use:

    datetime.utcfromtimestamp(ts).replace(tzinfo=timezone.utc)

This will be more efficient. You don't need `make_aware` here because UTC doesn't have DST.

Sure, this is a micro-optimization, but I like avoiding overhead at the lower levels ;-)